app/models/post.rb

```diff
@@ -3,6 +3,11 @@ class Post < ApplicationRecord
   has_many :comments, dependent: :destroy
   has_many :likes, dependent: :destroy
 
+  validates :Title, presence: true, length: { maximum: 250 }
+  validates :Text, presence: true
+  validates :CommentsCounter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+  validates :LikesCounter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+
```

Comment:
![image](https://user-images.githubusercontent.com/65068771/173169057-030c745a-2083-4d1d-8070-98f64a5386ec.png)

- [ ] Kindly the errors above are appearing because you capitalized the name of columns in your validation. Kindly use for example `title` and `likesCounter` instead of `Title`, and `likesCounter`

- [ ]  Also make sure you initialize the `postsCounter` to 0  the user is created so that you may apply you validation. Hint point_down